### PR TITLE
fix(mdc chips) Don't let MDC set aria-checked attribute on option chips

### PR DESCRIPTION
**Pull request by vanessanschmitt**

### CODE REVIEW

File: src/material-experimental/mdc-chips/chip.ts [hunk 1]
```diff
@@ -257,6 +257,12 @@ export class MatChip extends _MatChipMixinBase implements AfterContentInit, Afte
     },
     hasLeadingIcon: () => { return !!this.leadingIcon; },
     setAttr: (name: string, value: string) => {
+      // MDC is currently using this method to set aria-checked on choice and filter chips,
+      // which in the MDC templates have role="checkbox" and role="radio" respectively.
+      // We have role="option" on those chips instead; as such, we do not want aria-checked.
+      if (attr === 'aria-checked') {
```
Comment (crisbeto) [inline src/material-experimental/mdc-chips/chip.ts]: I think this is supposed to be `name`.

Comment (vanessanschmitt) [reply src/material-experimental/mdc-chips/chip.ts]: Whoops, bad copy paste from https://github.com/angular/components/pull/16528. Thank you!

File: src/material-experimental/mdc-chips/chip.ts [hunk 2]
```diff
@@ -257,6 +257,12 @@ export class MatChip extends _MatChipMixinBase implements AfterContentInit, Afte
     },
     hasLeadingIcon: () => { return !!this.leadingIcon; },
     setAttr: (name: string, value: string) => {
+      // MDC is currently using this method to set aria-checked on choice and filter chips,
+      // which in the MDC templates have role="checkbox" and role="radio" respectively.
+      // We have role="option" on those chips instead, so we do not want aria-checked.
+      if (name === 'aria-checked') {
```
Comment (jelbourn) [inline src/material-experimental/mdc-chips/chip.ts]: Could you file an issue on the MDC repo about this? I would consider it overzealous.

Comment (vanessanschmitt) [reply src/material-experimental/mdc-chips/chip.ts]: Done. https://github.com/material-components/material-components-web/issues/4910

**Pull request merged**